### PR TITLE
metric_schema: add pd metric tables

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
Add PD metric tables:

query example:
```sql
mysql>source ./total.sql
+-------------------------------------+----------------------------+----------------------------+----------------------------+----------------------------+----------------+
| metric                              | ref_time                   | sum_ref_value              | time                       | sum_value                  | ratio          |
+-------------------------------------+----------------------------+----------------------------+----------------------------+----------------------------+----------------+
| etcd_wal_fsync_duration             | 2019-12-30 23:45:00.001000 |          0.119044380952    | 2019-12-31 00:00:00.001000 |          0.352033809524    | 2.95716443487  |
| size_amplification                  | 2019-12-30 23:45:00.001000 |          4.96540153642     | 2019-12-31 00:00:00.001000 |          5.73616035014     | 1.15522587812  |
| pd_scheduler_region_heartbeat       | 2019-12-30 23:45:00.001000 |        272.004622846       | 2019-12-31 00:00:00.001000 |        293.335585662       | 1.07842132458  |
| pd_balance_scheduler_status         | 2019-12-30 23:45:00.001000 |      48060.2774711         | 2019-12-31 00:00:00.001000 |      49377.1210429         | 1.0273998329   |
| pd_checker_event_count              | 2019-12-30 23:45:00.001000 |     561845.397871          | 2019-12-31 00:00:00.001000 |     575976.107023          | 1.02515052932  |
| pd_server_etcd_state                | 2019-12-30 23:45:00.001000 |     360664.0               | 2019-12-31 00:00:00.001000 |     368730.0               | 1.02236430584  |
| pd_region_syncer_status             | 2019-12-30 23:45:00.001000 |        460.0               | 2019-12-31 00:00:00.001000 |        470.0               | 1.02173913043  |
| pd_schedule_filter                  | 2019-12-30 23:45:00.001000 |     143916.705809          | 2019-12-31 00:00:00.001000 |     146932.680363          | 1.02095638958  |
| region_label_isolation_level        | 2019-12-30 23:45:00.001000 |        245.0               | 2019-12-31 00:00:00.001000 |        250.0               | 1.02040816327  |
| region_health                       | 2019-12-30 23:45:00.001000 |        465.0               | 2019-12-31 00:00:00.001000 |        470.0               | 1.01075268817  |
| store_available_ratio               | 2019-12-30 23:45:00.001000 |          2.96249335908     | 2019-12-31 00:00:00.001000 |          2.97069698961     | 1.00276916419  |
| pd_scheduler_store_status           | 2019-12-30 23:45:00.001000 |          6.48035567978e+12 | 2019-12-31 00:00:00.001000 |          6.48056187512e+12 | 1.00003181852  |
| normal_stores                       | 2019-12-30 23:45:00.001000 |         10.0               | 2019-12-31 00:00:00.001000 |         10.0               | 1.0            |
| pd_scheduler_status                 | 2019-12-30 23:45:00.001000 |         40.0               | 2019-12-31 00:00:00.001000 |         40.0               | 1.0            |
| pd_scheduler_config                 | 2019-12-30 23:45:00.001000 |    2021934.0               | 2019-12-31 00:00:00.001000 |    2021934.0               | 1.0            |
| pd_role                             | 2019-12-30 23:45:00.001000 |         10.0               | 2019-12-31 00:00:00.001000 |         10.0               | 1.0            |
| pd_scheduler_balance_leader         | 2019-12-30 23:45:00.001000 |       2400.01067212        | 2019-12-31 00:00:00.001000 |       2399.98933926        | 0.999991111348 |
| pd_handle_transactions_rate         | 2019-12-30 23:45:00.001000 |          3.33334815572     | 2019-12-31 00:00:00.001000 |          3.33331852675     | 0.999991111348 |
| pd_cluster_status                   | 2019-12-30 23:45:00.001000 |          4.99145455186e+12 | 2019-12-31 00:00:00.001000 |          4.99137429223e+12 | 0.999983920594 |
| pd_handle_requests_duration         | 2019-12-30 23:45:00.001000 |          0.00491929878481  | 2019-12-31 00:00:00.001000 |          0.0049110678882   | 0.998326815067 |
| pd_grpc_completed_commands_rate     | 2019-12-30 23:45:00.001000 |         20.7334183188      | 2019-12-31 00:00:00.001000 |         20.5554454832      | 0.991416136361 |
| etcd_disk_wal_fsync_rate            | 2019-12-30 23:45:00.001000 |        300.002104363       | 2019-12-31 00:00:00.001000 |        267.998726593       | 0.893322822392 |
| pd_handle_requests_duration_avg     | 2019-12-30 23:45:00.001000 |          0.00134616089258  | 2019-12-31 00:00:00.001000 |          0.00115731010683  | 0.859711579208 |
| pd_handle_transactions_duration     | 2019-12-30 23:45:00.001000 |          0.0108225         | 2019-12-31 00:00:00.001000 |          0.0085575         | 0.790713790714 |
| pd_grpc_completed_commands_duration | 2019-12-30 23:45:00.001000 |          1.1722            | 2019-12-31 00:00:00.001000 |          0.8935            | 0.762241938236 |
| pd_hotspot_status                   | 2019-12-30 23:45:00.001000 | 3650975809.72              | 2019-12-31 00:00:00.001000 | 2193871506.17              | 0.600900038923 |
| pd_operator_finish_duration         | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| pd_operator_step_finish_duration    | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| pd_schedule_operator                | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| abnormal_stores                     | 2019-12-30 23:45:00.001000 |          0.0               | 2019-12-31 00:00:00.001000 |          0.0               | <null>         |
| pd_scheduler_tolerant_resource      | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| pd_scheduler_balance_direction      | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| label_distribution                  | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| pd_balance_region_movement          | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| pd_region_heartbeat_latency         | 2019-12-30 23:45:00.001000 |          0.0               | 2019-12-31 00:00:00.001000 |          0.0               | <null>         |
| pd_cluster_metadata                 | 2019-12-30 23:45:00.001000 |          0.0               | 2019-12-31 00:00:00.001000 |          0.0               | <null>         |
| pd_scheduler_op_influence           | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| pd_peer_round_trip_time_seconds     | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
| pd_schedule_store_limit             | <null>                     | <null>                     | <null>                     | <null>                     | <null>         |
+-------------------------------------+----------------------------+----------------------------+----------------------------+----------------------------+----------------+

```

### Check List 

Tests 

 - Unit test
 - Manual test (add detailed scripts or steps below)

### CODE REVIEW

File: infoschema/metric_schema_test.go [hunk 1]
```diff
@@ -37,12 +37,6 @@ func (s *inspectionSuite) TestMetricSchemaDef(c *C) {
 		} else {
 			c.Assert(def.Quantile == 0, IsTrue, Commentf("metric table %v has quantile, but doesn't contain $QUANTILE in promQL ", name))
 		}
-		if strings.Contains(def.PromQL, "$LABEL_CONDITIONS") {
-			c.Assert(len(def.Labels) > 0, IsTrue, Commentf("the labels of metric table %v should not be nil", name))
-		} else {
-			c.Assert(len(def.Labels) == 0, IsTrue, Commentf("metric table %v has labels, but doesn't contain $LABEL_CONDITIONS in promQL", name))
-		}
-
```
Comment (djshow832) [inline infoschema/metric_schema_test.go]: No testcases these newly added metrics? At least check whether the syntax is right.

Comment (crazycs520) [reply infoschema/metric_schema_test.go]: The removed test is not for the `syntax` check. 
`syntax` test needs to use `Prometheus` parser. So currently I only use manual tests to check the syntax.

Comment (crazycs520) [reply infoschema/metric_schema_test.go]: I know how to test it now, no need to remove this test.

**Pull request merged**